### ISSUE TITLE
Stops Responding On Loading Screen

### ISSUE THREAD

**Issue by babaliaris** [viewizard/astromenace#25]
## Specifications

Laptop With Full Specs: [Lifebook s760](https://www.cnet.com/products/fujitsu-lifebook-s760-13-3-core-i5-520m-windows-7-pro-4-gb-ram-160-gb-hdd-series/)
Operating System: Arch Linux 
KDE Plasma Version: 5.18.4
KDE Frameworks Version: 5.68.0
Qt Version: 5.14.1
Kernel-Version: 5.5.13-arch2-1
OS Type: 64-bit
Processors: 4 × Intel® Core™ i5 CPU M 520 @ 2.40GHz
Memory: 3.6 GiB of RAM
GPU Acceleration: NONE
GCC Version: GCC (Arch Linux 9.3.0-1) 9.3.0

  - Game Version:
 [Your latest commit when I cloned the master branch](https://github.com/viewizard/astromenace/commit/647f9017a1633e97c21511aed0c382fabf652450)

## Configuration Files
[config.txt](https://github.com/viewizard/astromenace/files/4432929/config.txt)
[PilotProfiles_1.4.txt](https://github.com/viewizard/astromenace/files/4432930/PilotProfiles_1.4.txt)

## Logging File
[log.txt](https://github.com/viewizard/astromenace/files/4432932/log.txt)

After Opening the game, I see the loading screen. After the loading finishes the music stops and the game stuck in the loading screen. If I use alt+f4 to kill the application, I get a Not Responding error from KDE plasma and I just click kill the application.

![IMG_20200405_070554](https://user-images.githubusercontent.com/17205585/78466656-581f9880-770c-11ea-88be-e4e0b401af27.jpg)

![IMG_20200405_070658](https://user-images.githubusercontent.com/17205585/78466678-aaf95000-770c-11ea-90fa-1f716d6225fd.jpg)

**Comment by babaliaris** [comment]
I found the problem. My CPU does not support the minimum requirements for your game. Because I saw that you are using SDL2, I thought the game would be software-based without GPU acceleration but you are probably using **OpenGL core profile at least version 3.3**, am I right? I used the [LIBGL_ALWAYS_SOFTWARE=1](https://www.mesa3d.org/envvars.html) to test it out and now the game works normally but with huge FPS drop (well that was expected).
